packages/runtime/container-runtime/src/summary/documentSchema.ts

```diff
@@ -59,26 +59,40 @@ export type IdCompressorMode = "on" | "delayed" | undefined;
  * @internal
  */
 export interface IDocumentSchema {
-	// version that describes how data is stored in this structure.
-	// If runtime sees a version it does not understand, it should immediately fail and not
-	// attempt to interpret any further data.
+	/**
+	 * Describes how data needed to understand the schema is stored in this structure.
+	 * If runtime sees a version it does not understand, it should immediately fail and not
+	 * attempt to interpret any further data.
+	 */
 	version: number;
 
-	// Sequence number when this schema became active.
+	/**
+	 * Sequence number when this schema became active.
+	 */
 	refSeq: number;
 
+	/**
+	 * Runtime configurations that affect the document schema. Other clients must understand these
+	 * properties to be able to open the document.
+	 */
 	runtime: Record<string, DocumentSchemaValueType>;
 }
-
 /**
  * Content of the type=ContainerMessageType.DocumentSchemaChange ops.
  * The meaning of refSeq field is different in such messages (compared to other usages of IDocumentSchemaCurrent)
```

Comment:
I think we may just want to delete this line, I don't see the difference.  I'll come back and comment again to confirm or recant after reading through the PR/code again